testsuite/cluster-test/src/experiments/reconfiguration_test.rs

```diff
@@ -19,7 +19,7 @@ use diem_types::{
     account_address::AccountAddress,
     chain_id::ChainId,
     ledger_info::LedgerInfoWithSignatures,
-    on_chain_config::{ConsensusConfigV2, OnChainConsensusConfig},
+    on_chain_config::{ConsensusConfigV1, OnChainConsensusConfig},
```

Comment:
The code in this file seems unrelated to the PR?

Reply:
ops, maybe rebase issue

Reply:
fixed